components/DragAndDropInput.vue

```diff
@@ -169,13 +169,13 @@ const handleKeydown = (e: KeyboardEvent) => {
 </template>
 
 <style lang="scss" scoped>
-.wrapper {
+.drag-and-drop {
```

Comment:
I like this class naming 👌 
And thanks for the info of declaring SCSS classes - to not nest them on first level 🙌